runners/spark/src/test/java/org/apache/beam/runners/spark/metrics/SparkBeamMetricTest.java

```diff
@@ -35,7 +35,8 @@ public void testRenderName() {
                 "myStep.one.two(three)", MetricName.named("myNameSpace//", "myName()")),
             123,
             456);
-    String renderedName = new SparkBeamMetric().renderName(metricResult);
+    new SparkBeamMetric();
```

Comment:
Why is instantiating an unused `SparkBeamMetric` necessary?

Reply:
It's not necessary, sorry I've removed it.